install/v2plugin/startcontiv.sh

```diff
@@ -4,125 +4,111 @@
 # run a cluster store like etcd or consul
 
 set -e
+echo "INFO: Starting contiv net with ARGS:"
+echo "$@"
+echo "INFO: Starting contiv net with ENV:"
+/usr/bin/env | grep CONTIV_
+
+if [ -z "$CONTIV_ROLE" ]; then
+    echo "CRITICAL: ENV CONTIV_ROLE must be set"
+    echo "CRITICAL: Unknown contiv role"
+    exit 1
+elif [ "$CONTIV_ROLE" != "netmaster" ] && [ "$CONTIV_ROLE" != "netplugin" ]; then
+    echo "CRITICAL: ENV CONTIV_ROLE must be in [netmaster, netplugin]"
+    echo "CRITICAL: Unknown contiv role"
+    exit 1
+fi
 
-if [ "$log_dir" == "" ]; then
-    log_dir="/var/log/contiv"
+# setting up logs
+if [ ! -z "$CONTIV_LOG_DIR" ]; then
```

Comment:
`-n` ?

Reply:
isn't ` ! -z ""` == `-n ""` ? is it a concern ?

Reply:
just a minor nitpick... you can avoid negation by using the opposing operator

Reply:
```
[chrisplo:~] $ 
$ CONTIV_LOG_DIR=foo
[chrisplo:~] $ 
$ if [ ! -z "$CONTIV_LOG_DIR" ]; then echo setting log dir; fi
setting log dir
[chrisplo:~] $ 
$ unset CONTIV_LOG_DIR
[chrisplo:~] $ 
$ if [ ! -z "$CONTIV_LOG_DIR" ]; then echo setting log dir; fi
[chrisplo:~] $ 
```
the logic is wrong I think, drop the `!`